src/esn.calendar.libs/app/components/time-select/time-select.component.js

```diff
@@ -9,6 +9,7 @@ angular.module('esn.calendar.libs')
       date: '<',
       onTimeChange: '=',
       locale: '@',
-      timeFormat: '@'
+      timeFormat: '@',
+      disabled: '=?'
```

Comment:
just for understanding, what is the meaning of `=?`, often I see `=` without `?`

Reply:
The use of = is wrong here cause it makes the binding bidirectional, meaning the component can change the property of the patent too.

It should have been @ instead to just readonly the value.

The ? Is used to prevent angular throwing an exception (NON_ASSIGNABLE_MODEL_EXPRESSION) when the parent scope.

I wanted to add the disabled as an optional property so that later on when we use the component in something else we don't have to provide the disabled property when we don't have to, like in the event time change  proposal.

So my bad it should have been @?, thanks and I'll fix it once Im in office.